src/main/java/com/ifactory/press/db/solr/processor/MultiSuggesterProcessor.java

```diff
@@ -31,7 +31,7 @@ public void processAdd(AddUpdateCommand cmd) throws IOException {
             suggester.add(doc, cmd.getReq().getSearcher());
         }
         if (next != null) {
-            next.processAdd(cmd);
+            next.processAdd(cmd);  // rivey this is the call above FieldMerging processAdd
```

Comment:
In this configuration, yes `next` is `FieldMergingProcessor`, but change the config (even just re-order it) and this will change. This appears to just be chaining of processors in an [updateRequestProcessorChaing](https://lucene.apache.org/solr/guide/6_6/update-request-processors.html#UpdateRequestProcessors-CustomUpdateRequestProcessorChain).

Reply:
Yes. Noted